packages/gatsby/cache-dir/page-renderer.js

```diff
@@ -119,7 +119,8 @@ class PageRenderer extends React.Component {
   }
 
   render() {
-    if (!this.state.pageResources) return null
+    if (!(this.state.pageResources && this.state.pageResources.json))
```

Comment:
was this tested in `develop`? I think in develop data comes from props passed by `json-store.js` and not from state

Reply:
I haven't explicitly tested this in develop yet - however, I think it should be fine since it was working previously with just the `!this.state.pageResources` check. I'll test properly once I've fixed the problems I'm currently working on.